include/rewd_controllers/MoveUntilTouchTopicController.hpp

```diff
@@ -0,0 +1,116 @@
+#ifndef REWD_CONTROLLERS_MOVEUNTILTOUCHTOPICCONTROLLER_HPP_
```

Comment:
Well, so it seems the `rewd_controllers` in general miss this kind of documentation (expect the base class where we added one line in the last PR). I'd suggest that this is a bigger issue which should be addressed for all `rewd_controllers` at the same time and which we should keep consistent.

So: Another PR to add documentation to all controllers, or start in this PR with this specific controller?

Reply:
Also, since the question was `where do we document`, I personally like files with example configurations as documentation, so that I can just copy-paste a working example and modify it as necessary.

Reply:
I agree. Let's do this in another PR, since we're trying to get this merged 😄

Reply:
Sounds good. Issue here: #23